include/kimera_vio_ros/RosBagDataProvider.h

```diff
@@ -1,85 +0,0 @@
-/**
- * @file   rosbag-data-source.h
- * @brief  ROS wrapper
- * @author Yun Chang
- * @author Antoni Rosinol
- */
-
-#pragma once
```

Comment:
why deleting rosbag parsing? :cry:

Reply:
Still WIP. I just removed it temporarily so it wouldn't mess with my IDE's code model. Though I'd like to reduce the complexity of the main functions. I'm thinking the online and bag parsing deserve to be separate nodes, rather than adding all this muxing logic in c++ mains.

Reply:
It makes sense to me, as long as we don't duplicate code.